Source/WebKit/UIProcess/Cocoa/TextExtraction/WKWebView+TextExtraction.swift

```diff
@@ -104,8 +107,12 @@ extension WKWebView {
             let coordinator = IntelligenceCollectionCoordinator.shared
             let collector = coordinator.createCollector(remoteContextWrapper: remoteContextWrapper)
 
-            if let item = await _requestTextExtraction(visibleRect) {
-                collector.collect(createIntelligenceElement(item: item))
+            var configuration = _WKTextExtractionConfiguration()
```

Comment:
this can be `let` (I think) (since it's a class so setting its properties doesn't really "count" as mutating it)

Reply:
Good catch, fixed.